sklearn/decomposition/_truncated_svd.py

```diff
@@ -185,8 +185,14 @@ def fit_transform(self, X, y=None):
 
         self.components_ = VT
 
+        # For consistent .fit_transform = .fit.transform, despite approx
```

Comment:
Could you elaborate a bit more. "As an SVD approximation for X is used, X@V is not the same as U@Sigma ..."